.github/workflows/eerelease.yml

```diff
@@ -0,0 +1,124 @@
+name: Release Ansible Execution Environment
+on:
+  workflow_dispatch:
+    inputs:
+      ee-name:
+        description: >-
+          EE name if it is community-ee-base or community-ee-minimal. Example: community-ee-base
+        required: true
+      ansible-core-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+      ee-version:
+        description: >-
+          EE Release Version. Example: 1
+        required: true
+      tag:
```

Comment:
How about changing this option name to `tag_as_latest`? That's a lot more expressive than simply `tag`.

Reply:
Updated it.